content/docs/sidebar.json

```diff
@@ -135,6 +135,12 @@
         ]
       },
       "experiment-management",
+      {
+        "slug": "checkpoints",
+        "tutorials": {
+          "katacoda": "https://katacoda.com/dvc/courses/get-started/checkpoints"
+        }
+      },
```

Comment:
You can preview the deployment now: https://dvc-org-mm-checkpoints--lv46zp.herokuapp.com/doc/user-guide/checkpoints! It looks like the navigation here needs to be modified since it's not showing up as a subsection of experiment management now.

Reply:
Just finished that round of updates! Let me know what y'all think! @dberenbaum @dmpetrov @jorgeorpinel